src/main/java/com/ubiquitech/leaveTrack/domain/Request.java

```diff
@@ -39,7 +39,7 @@
     private LocalDate endDate;
     private String comment;
     @NotEmpty
-    private String state;
+    private String state; /* should this not be an enum i.e it can only have certain values? */
```

Comment:
A "state" field is a good candidate for a java enum - it can only have certain values.